src/app/components/form/Input/index.tsx

```diff
@@ -38,10 +38,12 @@ export default function Input({
       name={name}
       id={id}
       className={classNames(
-        "block w-full placeholder-gray-500 dark:placeholder-neutral-600 dark:text-white",
-        !suffix && !endAdornment
-          ? `${outerStyles} focus:ring-orange-bitcoin focus:border-orange-bitcoin focus:dark:border-orange-bitcoin focus:ring-1`
-          : "pr-0 border-0 focus:ring-0 bg-transparent"
```

Comment:
Instead of doing the below changes, here you can:

- Modify `outerStyles` to remove unnecessary background and text styling (I wonder how it reached there)
```js
const outerStyles = "rounded-md border border-gray-300 dark:border-neutral-800 transition duration-300";
```
- Remove `bg-transparent` from line 44 (as `suffix` and `endAdornment` don't influence the background and text)
- Use `disabled` separately to control the background and text styling wholly, instead of `suffix` and `endAdornment`
```js
//...
!suffix && !endAdornment
  ? `${outerStyles} focus:ring-orange-bitcoin focus:border-orange-bitcoin focus:dark:border-orange-bitcoin focus:ring-1`
  : "pr-0 border-0 focus:ring-0",
disabled
  ? "bg-gray-50 dark:bg-surface-01dp text-gray-500 dark:text-neutral-500"
  : "bg-white dark:bg-black dark:text-white"
```

Reply:
Thank you for suggesting changes, I will do the mentioned changes